task/06-conditions-n-loops-tasks.js

```diff
@@ -166,7 +192,30 @@ function isInsideCircle(circle, point) {
  *   'entente' => null
  */
 function findFirstSingleChar(str) {
-    throw new Error('Not implemented');
+    let searchChar;
+    let result = null;
+    let newStr = null;
+
+    OUTER_LOOP: for (let i = 0; i < str.length; i++)
+    {
+        searchChar = str[i];
+        newStr = str.replace(searchChar, '');        
+
+        INNER_LOOP: for (let j = 0; j < newStr.length; j++)
+        {   
+            if (searchChar == newStr[j])
+            {
+                break INNER_LOOP;
+            }
+            if (j + 1 == newStr.length)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"